### ISSUE TITLE
Black output with Lora

### ISSUE THREAD

**Issue by popovidis** [tdrussell/diffusion-pipe#30]
Hey there, I have ~90 images (1024 width only, some are 1024 height but not always). I managed to get through the epochs all the way to 10 epochs, the workflow outputs black video. Am I doing something wrong in my workflow?

[black_output.json](https://github.com/user-attachments/files/18223026/black_output.json)

**Comment by tdrussell** [comment]
I haven't used video-to-video, but nothing stands out to me as being obviously wrong. A few things to check:
1. Without the LoRA, everything works correctly, right?
2. What about other LoRAs, do they work? There are several on Civit you can find.
3. What were your training loss values? Did they decrease as training progressed? You can use Tensorboard to see graphs of the various metrics.

**Comment by popovidis** [comment]
1. Yes
2. I have tried https://civitai.com/models/1029279/makima-hunyuan-character (their workflow looks weird because you can't attach "lora loader model only" since those are not compatible in the current version of comfyui node, picture from civit attached) It is also outputting black.
![image](https://github.com/user-attachments/assets/4853e3cd-9fcb-44ce-a7e6-3135cbf98111)
3. It went from ~0.07 ish to ~0.04 after 2 epochs. I simply am running the train command, no idea how to use tensorboard.

**Comment by tdrussell** [comment]
If other loras are giving the same black output then the problem is with your ComfyUI workflow. I can't really help much with your specific video-to-video workflow. If you are following some example, and known-working loras from civit are giving black outputs, then you should raise some issue with ComfyUI or the HunyuanVideo wrapper extension (depending on which nodes you're using).

**Comment by popovidis** [comment]
Just reran the example workflow from the wrapper. Also black output. Something went completely wrong I guess. Will be testing it more tomorrow. I guess its something wrong with my Comfy after all 😅

**Comment by popovidis** [comment]
https://github.com/kijai/ComfyUI-HunyuanVideoWrapper/issues/137

its probably this again... the container has 2.4.0 torch default...
